src/lib.rs

```diff
@@ -48,24 +48,24 @@
 //!
 //! ### Examples
 //!
-//! ```edition2018
+//! ```edition2021
```

Comment:
I think this can just be removed everywhere.
```suggestion
//! ```
```